### PR TITLE
Azd package details

### PR DESCRIPTION
**Pull request by alexwolfmsft**

### CODE REVIEW

**Comment by weikanglim** [comment]
@alexwolfmsft Thanks for making the change here! Do we have an issue tracking this? I very much agree with the user sentiment here that azd isn't providing enough details on `azd package`. Documenting this in the CLI help is the first step, but I could imagine that we could also think about broadening the feature + documentation here overall.

**Comment by alexwolfmsft** [comment]
> @alexwolfmsft Thanks for making the change here! Do we have an issue tracking this? I very much agree with the user sentiment here that azd isn't providing enough details on `azd package`. Documenting this in the CLI help is the first step, but I could imagine that we could also think about broadening the feature + documentation here overall.

Thanks @weikanglim  We had a feedback item come in through the docs system with the following:

"There is a lack of documentation around the '--from-package' argument on azd-deploy. I'm trying to work out how to get it to re-use existing build artifacts in a staged deploy pipeline."

https://dev.azure.com/msft-skilling/Content/_boards/board/t/Azure%20Developer%20CLI/Stories?System.AssignedTo=alexwolf%40microsoft.com&System.IterationPath=Content%5CSelenium%5CFY25Q2%5C12%20Dec&workitem=300397

I agree there is probably more we could do here in the broader docs, though I thought this was a simple first step. Do you know why the test is failing? I tried to troubleshoot it with a few different variations but I can't seem to get it to pass.

**Comment by JeffreyCA** [comment]
> > @alexwolfmsft Thanks for making the change here! Do we have an issue tracking this? I very much agree with the user sentiment here that azd isn't providing enough details on `azd package`. Documenting this in the CLI help is the first step, but I could imagine that we could also think about broadening the feature + documentation here overall.
> 
> Thanks @weikanglim We had a feedback item come in through the docs system with the following:
> 
> "There is a lack of documentation around the '--from-package' argument on azd-deploy. I'm trying to work out how to get it to re-use existing build artifacts in a staged deploy pipeline."
> 
> https://dev.azure.com/msft-skilling/Content/_boards/board/t/Azure%20Developer%20CLI/Stories?System.AssignedTo=alexwolf%40microsoft.com&System.IterationPath=Content%5CSelenium%5CFY25Q2%5C12%20Dec&workitem=300397
> 
> I agree there is probably more we could do here in the broader docs, though I thought this was a simple first step. Do you know why the test is failing? I tried to troubleshoot it with a few different variations but I can't seem to get it to pass.

I think it might be due to the `-h, --help : Gets help for deploy.` added to the .snap file but wasn't added in the implementation

File: cli/azd/cmd/testdata/TestUsage-azd-deploy.snap
Comment (weikanglim) [inline cli/azd/cmd/testdata/TestUsage-azd-deploy.snap:14]: Just curious, does this sound any better?

Deploys the packaged service located at the provided path. Supports zipped file packages (file path) or container images (image tag).

**Review by weikanglim — COMMENTED** [review]
This looks good to me, but I would kindly request another pair of eyes from @ellismg  or @wbreza before final approval

**Review by ellismg — APPROVED** [review]

**Pull request merged**
